Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2501,15 +2502,16 @@ def gen2021HiMix(fragment,howMuch):
                                        '--datatier':'MINIAODSIM,DQMIO',
                                        '--eventcontent':'MINIAODSIM,DQM',
                                        '-n':100,
+                                       '--era':'Run2_2018_pp_on_AA',
+                                       '--procModifiers':'run2_miniAOD_pp_on_AA_103X',
                                    },hiDefaults2018_ppReco,step3Up2015Defaults])
 steps['REMINIAODHI2018PPRECOMB']=merge([{'-s':'PAT,VALIDATION:@miniAODValidation,DQM:@miniAODDQM',
                                          '--datatier':'MINIAODSIM,DQMIO',
                                          '--eventcontent':'MINIAODSIM,DQM',
                                          '-n':100,
                                          '--era':'Run2_2018_pp_on_AA',
-                                         '--procModifiers':'genJetSubEvent',
+                                         '--procModifiers':'genJetSubEvent,run2_miniAOD_pp_on_AA_103X',
```

Comment:
@mandrenguyen , please check: is this additional procModifier ("run2_miniAOD_pp_on_AA_103X") really intended for `REMINIAODHI2018PPRECOMB`?
(I noticed it because, if I am not wrong, this showed up after the rebase, but it as not there before)

Reply:
Yes, this procModifier is the one that enables that cleaning of bad PF candidates (the point of this PR).  It should be enabled for the (re)miniAOD workflows, but not for the overall pp_on_AA eras.   The "MB" version of the reMiniAOD relval workflow was introduced in #31698, the PR that conflicted with this one, and which caused a rebase to be needed.  I'm not 100% sure of the sequence of events that led to this being reintroduced in the rebase, but I confirm that it is the intended behavior.